samplesheet_nobc_nodx.csv

```diff
@@ -1,3 +1,7 @@
-sample,fastq,barcode,genome,transcriptome
-K562_RUN1_REP3,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/demultiplexed/barcode01.fastq.gz,,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/hg19_KCMF1.fa,
-HEPG2_RUN3_REP5,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/demultiplexed/barcode02.fastq.gz,,,
+sample,input_file,barcode,genome,transcriptome,condition
+MCF7_directcDNA_replicate1,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/MCF7_directcDNA_replicate1.fastq.gz,,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/chr22_19431000-19528000.fa,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/chr22_19431000-19528000.gtf,MCF7
+MCF7_directcDNA_replicate3,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/MCF7_directcDNA_replicate3.fastq.gz,,,,MCF7
```

Comment:
You don't.need to specify the reference fasta and gtf for all entries?

Reply:
Thanks for the reminder! I have specified them.